<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCRS485_Calib</class>
 <widget class="QDialog" name="VCRS485_Calib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калибровка</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Калибровка датчиков напряжения</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Датчики для калибровки</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Volt_1">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>20</y>
      <width>18</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="Volt_2">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>20</y>
      <width>18</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>№1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>№2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Шкала датчика</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>265</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Верхняя точка напряжения, В*10000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>271</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нижняя точка напряжения, В*10000</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Dia">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>52</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>авто</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>максимум</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>минимум</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="up">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>92</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-800000</number>
    </property>
    <property name="maximum">
     <number>800000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dwn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>132</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-800000</number>
    </property>
    <property name="maximum">
     <number>800000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="diaWrite">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upWrite">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dwnWrite">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calibrate">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>170</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Калибровать</string>
    </property>
   </widget>
   <widget class="QLabel" name="dia_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>№1, №2</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>222</y>
     <width>265</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Время усреднения, секунд</string>
   </property>
  </widget>
  <widget class="QPushButton" name="avgWrite">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Записать</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="avg">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>222</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="passw">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>262</y>
     <width>220</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pswWrite">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Записать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пароль для записи</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
